packages/iov-bns-governance/types/index.d.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./constants";
+export * from "./governor";
+export * from "./proposals";
```

Comment:
Can we make the exported types explicit? It is important that the caller projects that's have more types available than they should use.